pkg/ee/metering/reconcile.go

```diff
@@ -59,7 +59,7 @@ const (
 )
 
 func getMeteringImage(overwriter registry.WithOverwriteFunc) string {
-	return overwriter(resources.RegistryQuay) + "/kubermatic/metering:v1.0.0"
+	return overwriter(resources.RegistryQuay) + "/kubermatic/metering:v1.0.1"
```

Comment:
Tag doesnt exist in https://github.com/kubermatic/metering/tags, what is this pointing to?

Reply:
if it's not released yet, why are we including it in a KKP release branch? :D

Reply:
Meanwhile released as discussed in slack.